tests/sentry/integrations/jira_server/test_integration.py

```diff
@@ -428,16 +440,108 @@ def get_priorities_callthrough_client(installation: JiraServerIntegration):
             priority_field = [field for field in fields if field["name"] == "priority"][0]
             assert priority_field == {
                 "default": "",
+                # Choices should exclude the low priority
```

Comment:
why?

Reply:
The priority scheme we match to the current project only has the `High` priority, see [here](https://github.com/getsentry/sentry/pull/59197/files#diff-63c86a8a3b908c4e19d76f3f9daf5141b2b6b65b93baa6ee81db838566d2014aR388-R390)